### DISCUSSION TITLE
Are there router outlets in nextjs

### DISCUSSION THREAD

**Discussion by RollsChris** [vercel/next.js#47519] [Help]
### Summary

I'm trying to reproduce partial/nested routes in nextjs. Does anyone have any good tutorials?

a good example is if i have a list of heros and i select one, i want to display the hero details in a outlet next to the list? 

In angular id use nested routes with router outlet on the page

Does that make sense?

Thanks

**Comment by lpknv** [comment]
This is the same question I had since I am currently looking for an angular "router-outlet" equivalent in nextjs, which does not seem to exist.
Although reading the next docs, there was nothing like a "router-outlet" or at least I couldn't find it.

Someone tho mentioned [parallel routes](https://nextjs.org/docs/app/building-your-application/routing/parallel-routes) and [intercepting routes](https://nextjs.org/docs/app/building-your-application/routing/intercepting-routes) could do a similar thing, but I did not succeed with that.

**Comment by igortas** [comment]
Same boat, how I can achieve with ```app``` folder, I need home route to redirect to some nested route like /path1/path2, but when the app redirect to /path1/path2 all code and layouts from path1 and path2 should be executed

**Reply by lpknv** [reply]
Any updates on this one?

**Comment by Pervej0** [comment]
I have solved the issue. Here is below my gitHub repo. You will find code inside dashboard [grouping route](https://nextjs.org/docs/app/building-your-application/routing/route-groups).

gitHub: https://github.com/Pervej0/project-management
